src/Monitors/IMUMonitor.cpp

```diff
@@ -43,6 +43,7 @@ void IMUMonitor::execute()
 #ifdef VERBOSE
         Serial.println("Turned on IMU");
 #endif
+        sfr::imu::init_mode = (uint16_t)sensor_init_mode_type::awaiting;
```

Comment:
What is the reasoning behind adding this line?

Reply:
![image](https://github.com/Alpha-CubeSat/oop-flight-code/assets/114702666/2686a37f-8791-4456-8325-ff9e7121ad32)
IMU_init only calls IMU.begin() if init_mode is in sensor_init_mode_type::awaiting. So without that line, the IMU didn't turn back on after it was turned off the first time (init_mode was stuck in sensor_init_mode_type::complete from the first IMU initialization).

Reply:
We have not tested this specific case (turning the camera back on after turning it off), as the camera only gets turned off after the photo has been written to the sd. So definitely plausible that this could be an issue if we wanted to restart the camera for some reason.